brave/brave-core
# New welcome flow for default search

- Change flow for how users select their default search engine. Gives user ability to use immediate dropdown rather than go through settings
- See https://github.com/brave/brave-ui/pull/472 for storyboard and new SVG change
- https://github.com/brave/brave-browser/issues/1548
## Submitter Checklist:
- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:
- Ensure dropdown works
- Ensure button is disabled if no search engine is selected 
- Ensure display text changes based on locale
- (default) should be appended to the region's default search provider
- Ensure 'Confirm' button still links to next page of on-boarding flow
- Ensure responsiveness on smaller screens
- Ensure new search engine selected becomes the default search engine in address bad

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
